ansible/roles/test/tasks/fast-reboot.yml

```diff
@@ -3,7 +3,14 @@
       reboot_limit: 30
   when: reboot_limit is not defined
 
+- name: set default values vnet variables
+  set_fact:
+      vnet: False
+      vnet_pkts: ''
+  when: (vnet is not defined) or (vnet_pkts is not defined)
```

Comment:
Should we provide default value inside `roles/test/files/ptftests/advanced-reboot.py`? I notice the arguments are not required in that file

Reply:
The variables need to be defined in 
[https://github.com/Azure/sonic-mgmt/blob/094c6f4cbc89c664549d6fed256b639c9aad6619/ansible/roles/test/tasks/ptf_runner_reboot.yml#L74](L74), otherwise we will get variable undefined error.

Reply:
Do you know a way to only pass vars parameter only if it's defined? 
Maybe ```default(omit)``` will work, need to give it a try

Reply:
@qiluo-msft I tried the ``` default(omit)``` option, It does not work for passing parameters to ptf.
We end up with a command like this (notice the omit placeholders): ```ptf --test-dir ptftests advanced-reboot.ReloadTest  --qlen=1000   --platform-dir ptftests  --platform remote  -t \"verbose=False;dut_username=\\\"admin\\\";dut_hostname=\\\"arc-switch1041\\\";reboot_li
mit_in_seconds=30;reboot_type=\\\"fast-reboot\\\";portchannel_ports_file=\\\"/tmp/portchannel_interfaces.json\\\";vlan_ports_file=\\\"/tmp/vlan_interfaces.json\\\";ports_file=\\\"/tmp/ports.json\\\";dut_mac='24:8a:07:cb:7b:40';
dut_vlan_ip='192.168.0.1';default_ip_range='192.168.0.0/16';vlan_ip_range='192.168.0.0/21';lo_v6_prefix='fc00:1::/64';arista_vms=\\\"['10.213.86.64','10.213.86.63','10.213.86.62','10.213.86.61']\\\";preboot_files='None';preboot
_oper='None';inboot_oper='None';nexthop_ips=None;allow_vlan_flooding='False';sniff_time_incr=60;setup_fdb_before_test=True;vnet=__omit_place_holder__c68d073ba9332d4a08603f82a8dbff1aefd6b987;vnet_pkts='__omit_place_holder__c68d0
73ba9332d4a08603f82a8dbff1aefd6b987'\"``` which breaks the ptf parameters.

Reply:
Can we merge as is?